src/components/Project-block/Project-tile/Project-tile.module.css

```diff
@@ -0,0 +1,65 @@
+.projectWrapper {
+    flex: 0 1 var(--basic-screen-width);
+    border: var(--blue) 4px solid;
+    overflow: hidden;
+    list-style-type: none;
+    border-radius: 15px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.projectHeader {
+    color: var(--yellow);
+    background-color: var(--blue);
+    width: 100%;
+    text-align: center;
+    padding: 10px 40px;
+}
+
+.contentWrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+.pictureWrapper {
+    align-self: stretch;
+}
+
+.textWrapper {
+    flex: 1 1 340px;
+    padding: 40px 40px;
+    font-size: var(--fontsize-400);
+
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.listItemsProject {
+    list-style-position: inside;
+}
+
+.buttonWrapper {
```

Comment:
maybe add flex-wrap when there are more buttons than two or the text is very long